ir/src/constraints/graph.rs

```diff
@@ -98,7 +98,7 @@ impl AlgebraicGraph {
     // --- PUBLIC MUTATORS ------------------------------------------------------------------------
     /// Insert the operation and return its node index. If an identical node already exists, return
     /// that index instead.
-    pub(super) fn insert_op(&mut self, op: Operation) -> NodeIndex {
+    pub(crate) fn insert_op(&mut self, op: Operation) -> NodeIndex {
```

Comment:
nit: I've been wondering about changing this to `insert_node` or `insert_graph_node` as well. Wdyt?

Reply:
Changed it to `insert_node` since it seems like `graph` is already implied here.